common-token/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![deny(warnings)]
+#![no_std]
+
+use soroban_sdk::{Address, Env};
+
+pub mod balance;
+pub mod storage;
+
+pub fn verify_caller_is_pool(e: &Env) -> Address {
```

Comment:
I think, I would not expect any value (except `panic`) from the function named `verify_`. Can we rename or split it?